src/sql/workbench/contrib/tableDesigner/browser/tableDesignerTextEditor.ts

```diff
@@ -14,7 +21,27 @@ export class TableDesignerTextEditor implements DesignerTextEditor {
 	private _contentChangeEventEmitter: Emitter<string> = new Emitter<string>();
 	readonly onDidContentChange: Event<string> = this._contentChangeEventEmitter.event;
 
-	constructor(container: HTMLElement, @IInstantiationService instantiationService: IInstantiationService) {
+	private _editor: TableDesignerBaseTextEditor;
+	private _untitledTextEditorModel: UntitledTextEditorModel;
+	private _editorInput: UntitledTextEditorInput;
+	private _editorModel: ITextModel;
+
+	constructor(
+		private _container: HTMLElement,
+		@IInstantiationService private _instantiationService: IInstantiationService) {
+		this._editor = this._instantiationService.createInstance(TableDesignerBaseTextEditor);
+		this._editor.create(this._container);
+		this._editor.setVisible(true);
+		this._untitledTextEditorModel = this._instantiationService.createInstance(UntitledTextEditorModel, URI.from({ scheme: Schemas.untitled }), false, undefined, 'sql', undefined);
+		this._editorInput = this._instantiationService.createInstance(UntitledTextEditorInput, this._untitledTextEditorModel);
+		this._editor.setInput(this._editorInput, undefined, undefined);
+		this._editorInput.resolve().then((model) => {
```

Comment:
how do I set the content?

Reply:
in the set content function, updated now